.changeset/heavy-bags-wait.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/icons': patch
```

Comment:
suggestion: this might be a minor change since it's adding a new icon

Reply:
True! Updated.